### PR TITLE
Register satr14.is-a.dev

### PR DESCRIPTION
**Pull request by satr14washere**
Register satr14.is-a.dev with CNAME record pointing to sx-9.github.io.

### CODE REVIEW

File: domains/justtestingstuff.json [hunk 1]
```diff
@@ -0,0 +1,12 @@
+{
+        "owner": {
+           "username": "SX-9",
+           "email": "satriakrach@gmail.com",
+           "discord": "882595027132493864"
+        },
+    
+        "record": {
+            "TXT": "yes"
```
Comment (DEV-DIBSTER) [inline domains/justtestingstuff.json:9]: TXT record alone doesn't do anything.

File: domains/​​.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "SX-9",
+        "email": "satriakrach@gmail.com"
+    },
+    "record": {
+        "TXT": "​"
```
Comment (DEV-DIBSTER) [inline domains/​​.json:7]: Blank TXT record.

File: domains/​​.json
Comment (DEV-DIBSTER) [inline domains/​​.json:1]: Blank domain name.

**Review by DEV-DIBSTER — CHANGES_REQUESTED** [review]

**Comment by SX-9** [comment]
@DEV-DIBSTER all of those are changes i forgot to delete

**Comment by SX-9** [comment]
you can merge now

**Review by DEV-DIBSTER — APPROVED** [review]

**Pull request merged**